app/views/profiles/preview.html.slim

```diff
@@ -0,0 +1,87 @@
+#Head style='margin: auto; width: 90%; background-color: #eeeeee; text-align: center; font-family: Ubuntu;'
```

Comment:
why don't create a css/scss file for styling ?

Reply:
This is temporary, later this will be took over by template partials, and i did use scss for templates.

Reply:
This is temporary, later this will be took over by template partials, and i did use scss for templates.